pre_commit_hooks/yamlfmt

```diff
@@ -91,7 +120,8 @@ class Formatter:
             offset=kwargs.get("offset", DEFAULT_INDENT["offset"]),
             )
         yaml.top_level_colon_align = kwargs.get("colons", False)
-        yaml.explicit_start = kwargs.get("implicit_start", True)
+        yaml.explicit_start = kwargs.get("explicit_start", True)
+        yaml.explicit_end = kwargs.get("explicit_end", False)
```

Comment:
This `kwarg` should never be non-existent, but just in case I left the default of `False` here too.